md_test.py

```diff
@@ -1,27 +1,19 @@
 import MDAnalysis as mda
 
 # Load simulation results with a single line
-u = mda.Universe('data/pdb/1fuu_noPTM_complex.top', 'data/traj/1fuu_noPTM_10-20ns_100snap.trr')
+u = mda.Universe('01_input/top/1fuu_noPTM_complex.top', '01_input/traj/1fuu_noPTM_10-20ns_100snap.trr')
+# u = mda.Universe('1a3w_noPTM.nmd_traj.pdb.gz')
```

Comment:
The changes to this file are technically unrelated, but they update the test script to be able to run with the current structure.